oslabs-beta/Kubiquity
# Convert CSS to Sass

WHAT DOES THIS PR DO? 
1.  Convert CSS work to Sass. 
2. Remove local style imports; import all styles via the app stylesheet. 
3. Fix property names in Loading component file to remove console errors/warnings. 

HOW TO TEST THIS PR:
1. Run the following command: `gh pr checkout 24 && npm install && npm run dev`
2. Verify that the app basically looks the same. 

ADDITIONAL NOTES: 
1. I removed the top shadow because Victor made fun of it. :,(
